test/Uds-Config-Tool/Functional Tests/funcTest_UtilityFunctions.py

```diff
@@ -1,19 +1,20 @@
-from uds.uds_config_tool.UtilityFunctions import (
-    getLongName,
-    getParamWithSemantic,
-    getPositiveResponse,
-    getSdgsData,
-    getSdgsDataItem,
-    getServiceIdFromDiagService,
-    getShortName,
-)
+from pathlib import Path
+
+from uds.uds_config_tool.UtilityFunctions import (getLongName,
+                                                  getParamWithSemantic,
+                                                  getPositiveResponse,
+                                                  getSdgsData, getSdgsDataItem,
+                                                  getServiceIdFromDiagService,
+                                                  getShortName)
 
 if __name__ == "__main__":
     import xml.etree.ElementTree as ET
 
+    here = Path(__file__).parent
```

Comment:
not in use right ?

Reply:
It is in use two lines down.:)

Reply:
as cody said. I just changed it so it is more obvious what is the name of the file and what the path